device/barefoot/x86_64-accton_as9516_32d-r0/platform.json

```diff
@@ -0,0 +1,196 @@
+{
+    "interfaces": {
+        "Ethernet0": {
+            "index": "1,1,1,1,1,1,1,1",
+            "lanes": "0,1,2,3,4,5,6,7",
+            "alias_at_lanes": "Ethernet0,Ethernet1,Ethernet2,Ethernet3,Ethernet4,Ethernet5,Ethernet6,Ethernet7",
+            "breakout_modes": "1x400G[200G],2x200G[100G,40G],4x100G[50G],1x100G(4),1x25G(2)+1x25G(2)"
```

Comment:
Instead of `1x100G(4),1x25G(2)+1x25G(2)` I believe should be `2x100G[40G],8x25G[10G],1x100G[40G](4)+4x25G[10G](4),4x25G[10G](4)+1x100G[40G](4)`